client/src/components/JobList.tsx

```diff
@@ -20,15 +36,50 @@ export default function JobList() {
     loadJobs();
   }, []);
 
```

Comment:
If you checked that `jobs` is defined up here and displayed a Loading... message, you could avoid the `jobs?.` optional chaining below. It would also probably produce a nicer display for the user.

Reply:
I've been meaning to set up a loading state. I didn't think about the fact that it could eliminate the optional chaining.